### DISCUSSION TITLE
long comment pr bubbles

### DISCUSSION THREAD

**Discussion by ferologics** [community/community#40568] [Projects and Issues]
![image](https://user-images.githubusercontent.com/5576161/205129726-bedb4c9d-4a18-4abb-89c0-8aae41a926ff.png)

**Comment by ossanna16** [comment]
Hi there @ferologics and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
